ding/policy/a2c.py

```diff
@@ -14,68 +15,90 @@
         type='a2c',
-        # (bool) Whether to use cuda for network.
+        # (bool) Flag to enable CUDA for model computation.
         cuda=False,
-        # (bool) whether use on-policy training pipeline(behaviour policy and training policy are the same)
-        on_policy=True,  # for a2c strictly on policy algorithm, this line should not be seen by users
+        # (bool) Flag for using on-policy training (training policy is the same as the behavior policy).
+        on_policy=True,
+        # (bool) Flag for enabling priority experience replay. Must be False when priority_IS_weight is False.
         priority=False,
-        # (bool) Whether use Importance Sampling Weight to correct biased update. If True, priority must be True.
+        # (bool) Flag for using Importance Sampling weights to correct updates. Requires `priority` to be True.
         priority_IS_weight=False,
-        # (str) Which kind of action space used in PPOPolicy, ['discrete', 'continuous']
+        # (str) Type of action space used in the policy, with valid options ['discrete', 'continuous'].
         action_space='discrete',
+        # learn_mode configuration
         learn=dict(
-
-            # (int) for a2c, update_per_collect must be 1.
-            update_per_collect=1,  # fixed value, this line should not be modified by users
+            # (int) Number of updates per data collection. A2C requires this to be set to 1.
+            update_per_collect=1,
+            # (int) Batch size for learning.
             batch_size=64,
+            # (float) Learning rate for optimizer.
             learning_rate=0.001,
-            # (List[float])
+            # (tuple[float, float]) Coefficients used for computing running averages of gradient and its square.
             betas=(0.9, 0.999),
-            # (float)
+            # (float) Term added to the denominator to improve numerical stability in optimizer.
             eps=1e-8,
-            # (float)
+            # (float) Maximum norm for gradients.
             grad_norm=0.5,
-            # ==============================================================
-            # The following configs is algorithm-specific
-            # ==============================================================
-            # (float) loss weight of the value network, the weight of policy network is set to 1
+            # (float) Scaling factor for value network loss relative to policy network loss.
             value_weight=0.5,
-            # (float) loss weight of the entropy regularization, the weight of policy network is set to 1
+            # (float) Weight of entropy regularization in the loss function.
             entropy_weight=0.01,
-            # (bool) Whether to normalize advantage. Default to False.
+            # (bool) Flag to enable normalization of advantages.
             adv_norm=False,
+            # (bool) Whether or not to consider "done" flags for episode termination.
```

Comment:
Maybe this should be changed like: "Flag to enable consideration of "done" flags for episode termination."

Reply:
polished now